src/containers/Tenant/Diagnostics/TenantOverview/Healthcheck/Healthcheck.scss

```diff
@@ -0,0 +1,110 @@
+@use '../../DetailedOverview/DetailedOverview.scss' as detailedOverview;
```

Comment:
It's better to move variables from `DetailedOverview.scss` to `src/styles/constants.scss`. It makes their usage more clear. They could be named like `--diagnostics-section-title-margin`